File: STTestTaskProject/Models/Staff.cs
using System;

namespace STTestTaskProject.Models
{
    public class Staff
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Surname { get; set; }
        public string Lastname { get; set; }
        public DateTime CreationDate { get; set; }
        public DateTime LastEditDate { get; set; }
        public DateTime EmploymentDate { get; set; }
        public Position Position { get; set; }
        public int PositionId { get; set; }
        public Department Department { get; set; }
        public int DepartmentId { get; set; }
    }
}

File: STTestTaskProject/ClientApp/src/models/Staff.ts
import {Department} from "./Department";
import {Position} from "./Position";

export class Staff {
  id: number;
  name: string;
  surname: string;
  lastname: string;
  creationDate: Date;
  lastEditDate: Date;
  employmentDate: Date;
  position: Position;
  department: Department;
  positionId: number;
  departmentId: number;
  constructor() {
    this.id = 0;
  }
}

